setup.py

```diff
@@ -148,6 +149,10 @@
                 "share/snapcraft/libraries",
                 ["libraries/" + x for x in os.listdir("libraries")],
             ),
+            (
+                "share/snapcraft/template-data",
+                ["templates/" + x for x in os.listdir("templates")],
```

Comment:
I wanted to get rid of this mechanism and do something a bit more pkg_resource driven as this fails when creating a source package to push over to pypi (unless you edit that other file https://github.com/snapcore/snapcraft/blob/master/MANIFEST.in). If possible, we should get away from that early.

Reply:
It's probably beyond the scope of this PR to change this, so I'll edit the manifest here, but yeah, let's change that.